network/delegate_packet_proxy.go

```diff
@@ -19,14 +19,14 @@ import (
 	"sync/atomic"
 )
 
-// DelegatePacketProxy is a PacketProxy that forwards calls (like NewSession) to another PacketProxy. To create a
-// DelegatePacketProxy with the default PacketProxy, use NewDelegatePacketProxy. To change the underlying PacketProxy,
-// use SetProxy.
+// DelegatePacketProxy is a [PacketProxy] that forwards calls (like NewSession) to another [PacketProxy]. To create a
+// DelegatePacketProxy with the default PacketProxy, use [NewDelegatePacketProxy]. To change the underlying
+// [PacketProxy], use SetProxy.
 //
-// Note: After changing the underlying PacketProxy, only new NewSession calls will be routed to the new PacketProxy.
-// Existing sessions will not be affected.
+// Note: After the underlying [PacketProxy] is changed, only new NewSession calls will be routed to the new
```

Comment:
should be: [New Session]

Reply:
interface method names should not be wrapped in `[]`